cdk.tf/vercel.json

```
@@ -180,6 +180,9 @@
       "destination": "https://registry.terraform.io/browse/providers"
     },
     {
+      "source": "/crash-reporting",
+      "destination": "https://developer.hashicorp.com/cdktf/telemetry#crash-reporting"
+     },
```

JSON seems to be invalid 😅 That's probably why Vercel failed.

```suggestion
     },
     {
```